### DISCUSSION TITLE
Cierre Semana XI

### DISCUSSION THREAD

**Discussion by NicoSgandurra** [CodeSystem2022/Primates-Tercer-Semestre#131] [Announcements]
El merge de todas las ramas para la clase 11 se llevará a cabo el sábado 01/07/23

**Comment by Reinoso98** [comment]
mi ejercicio de clase 11 ya quedo cargado

**Comment by taniachoquevillca28** [comment]
estoy trabajando en la clase 11

**Comment by mauricioluis85** [comment]
ya realice mi ejercicio 11

**Comment by taniachoquevillca28** [comment]
subi el ejercicio 11
